packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,240 @@
+          }
+          setCurrentInputValue(inputValue);
+        }}
+        // Selected Items in AutoComplete
+        selected={currentSelections}
+        onSelectChange={({ values }) => {
+          setCurrentSelections(values);
+        }}
+      />
+      <DropdownOverlay>
+        <ActionList>
+          {cities.map((city) => (
+            <ActionListItem
+              title={city}
+              value={city.toLowerCase()}
+            />
+          ))}
+        </ActionList>
+      </DropdownOverlay>
+    </Dropdown>
+  );
+};
+```
+
+</details>
+
+### In BottomSheet
+
+> **Warning**
+>
+> We will have to turn BottomSheetHeader (and maybe BaseHeader) to a slot to allow adding Input element there.
+
+**Open Question:** Should we also remove `leading` `trailing` etc props from Header? since same can be implemented inside the slot. This is what we follow for Footer.
+
+<img height="200" alt="image" src="https://github.com/razorpay/blade/assets/30949385/6867b09a-1ae2-4c4a-a977-e254b45b5282">
+
+```jsx
+<Dropdown selectionType="multiple">
+  <SelectInput label="Select Emails" />
+  <BottomSheet>
+    <BottomSheetHeader title="Do you want this report in an email?">
+      <AutoComplete label="Add receiver's email addresses" />
+    </BottomSheetHeader>
+    <BottomSheetBody>
+      <ActionList>
+        <ActionListItem title="Mumbai" value="mumbai" />
+        <ActionListItem title="Pune" value="pune" />
+        <ActionListItem title="Hyderabad" value="hyderabad" />
+      </ActionList>
+    </BottomSheetBody>
+  </BottomSheet>
+</Dropdown>
+```
+
+### With Tags Outside
+
+When possible selections are more than 11, it is advised to put tags outside of the Input.
+
+<img width="700" alt="image" src="https://github.com/razorpay/blade/assets/30949385/8906eb6c-fae0-4e99-bd3f-d7e0edc60537">
+
+We add `showTags={false}` to AutoComplete to remove tags from input. Then we can use Controlled state to show tags outside in the UI.
+
+If not tags, what to show inside AutoComplete :thinking:? Checkout this table to understand -
+
+| Code                                                            | Output                                                                                                                     |
+| --------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `<AutoComplete label="Status" showTags={false} />`              | <img width="220" alt="image" src="https://github.com/razorpay/blade/assets/30949385/c8a30845-812c-493f-a51f-97c233880960"> |
+| `<AutoComplete accessibilityLabel="Status" showTags={false} />` | <img width="200" alt="image" src="https://github.com/razorpay/blade/assets/30949385/83cf63d5-ccc4-4bb9-bf0d-ce91dc5c1b09"> |
+
+So when input has label, we show number of selected items.
+
+When input doesn't have a label, we show accessibilityLabel inside the input.
```

Comment:
this seems a bit unexpected, accessibilityLabel i think of as a invisible prop which enables screen readers to get more information about the current component. 

Now we are saying that if accessibilityLabel is provided we will render that text. 

Instead it makes sense that: 
- accessibilityLabel -> show "2 more"
- label -> show "Status"

Reply:
You mean having a `showLabel` prop?